### PR TITLE
Pull request #1

### PR DESCRIPTION
**Pull request by Blasco9**
In this branch we:
- Created html and css files
- Constructed the html structure
- Styled the page
- Fixed the stylelint errors

### CODE REVIEW

**Comment by shshamim63** [comment]
## Status :red_circle: [Changes Required] :red_circle: 
___
Hi @Blasco9  and @phfdonda :wave:,
Page looks almost like the original one. Does not contain any commented code. Awesome job :clap: .
But still there are some issues that are blocking me to approve this project.
___
#### Please follow the instructions below to complete this project.
___
### :spades: [Mandatory Changes] :spades: 
___
1. The project does not contain a well descriptive readme. you can enrich the readme by asking yourself some questions like->
....1. What have you done in this project?
....2. What are the technical tools you have used?
....3. Is there any live version availabe?
....4. Who are the contributors?
....5. How to communicated with the contributors?
2. Stickler is not functional. Can not see any .stickler.yml file on this repo. So please follow the instructions carefully described in this [link](https://github.com/microverseinc/linters-config/tree/master/css).
3. You were asked to check HTML markup with this validator https://validator.w3.org/#validate_by_upload. ([screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/html_css/section/validation_requirement.png) from Pathwright). I did that and have found some errors. So please resolve them.
4. I can see mode_module and package-lock.json. They are not needed in here. So please exclude them from this repo.
___
### Please fix the above mentioned issue and send another review request.
___
## :herb: Happy Coding :herb: 
### Shakhawat Hossain
[Linkedin](https://www.linkedin.com/in/shakhawathossainshamim/)

**Review by mosesogwo — APPROVED** [review]
### Hi team, you have done well in effecting the requested changes.

**However, you have neglected Github flow completely. You are working on both branches. Since this project has been reviewed before, I'll give approval and you can merge to master**.

**Make sure you understand Github flow properly before starting your next project. No work should appear on the master branch before your pull request is approved**

Please fix the alignment of the text in the sign-in button before merge: 
image

### Status: Approved!

**Comment by Blasco9** [comment]
Thanks for your feedback @mosesogwo!.
I have a question about the github flow, should we have added the linter files only to the development branch or was the problem the readme file that we updated?
Also neither my partner or me had problems with the button alignment, we don't know why you have it displayed like that, if you could tell us the screen size and the browser that you are using that would be really helpful.
Thanks!

**Pull request merged**
